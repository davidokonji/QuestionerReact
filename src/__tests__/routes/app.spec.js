import React from 'react';
import { shallow } from 'enzyme';
import Routes from '../../routes/App.routes';

describe('Routes', () => {
  it('should render routes jsx', () => {
    const wrapper = shallow(<Routes />);

    expect(wrapper).toMatchSnapshot();
  });
});
