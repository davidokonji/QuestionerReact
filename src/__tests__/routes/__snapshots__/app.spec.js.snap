// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should render routes jsx 1`] = `
<BrowserRouter>
  <Switch>
    <Route
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(LandingPage)",
          "type": [Function],
        }
      }
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/auth"
    />
    <Protected
      component={[Function]}
      path="/dashboard"
    />
    <Protected
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(SingleMeetup)",
          "type": [Function],
        }
      }
      exact={true}
      path="/meetup/:id"
    />
    <Protected
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(AdminDashboard)",
          "type": [Function],
        }
      }
      path="/admin"
    />
    <Protected
      component={
        Object {
          "$$typeof": Symbol(react.memo),
          "WrappedComponent": [Function],
          "compare": null,
          "displayName": "Connect(UserProfile)",
          "type": [Function],
        }
      }
      path="/user"
    />
    <Route
      component={[Function]}
    />
  </Switch>
</BrowserRouter>
`;
